apps/designer/app/shared/context.server.ts

```diff
@@ -0,0 +1,58 @@
+import type { AppContext } from "@webstudio-is/trpc-interface/server";
+import { z } from "zod";
+import * as cryptoJson from "./crypto/crypto-json.server";
+import { authenticator } from "~/services/auth.server";
+import { trpcClient } from "~/services/trpc.server";
+import { getMode } from "./router-utils/build-params";
+
+const ReadToken = z.object({
+  projectId: z.string(),
+});
+
+export type ReadToken = z.infer<typeof ReadToken>;
```

Comment:
if type like this will become project-wide global, should we  rather call it AuthReadToken?